src/Entity/EntityIdValue.php

```diff
@@ -103,6 +96,7 @@ public function getSortKey() {
 	 * @return self
 	 */
 	public function getValue() {
+		// TODO: this should be changed to return the EntityId.
```

Comment:
Because the actual value is the EntityId. But this method is pointless anyway. We should just delete it.